src/Cli/Configuration/DomainConfiguration.php

```diff
@@ -48,7 +48,23 @@ public function getConfigTreeBuilder()
                         ->ifTrue(function ($item) {
                             return !filter_var($item, FILTER_VALIDATE_EMAIL);
                         })
-                        ->thenInvalid('The email "%s" is not valid.')
+                        ->thenInvalid('The email %s is not valid.')
+                    ->end()
+                ->end()
+                ->scalarNode('key_type')
+                    ->info('Type of private key (RSA or EC).')
+                    ->defaultValue('RSA')
+                    ->beforeNormalization()
+                        ->ifString()
+                        ->then(function ($conf) {
+                            return \strtolower($conf);
+                        })
+                    ->end()
+                    ->validate()
+                        ->ifTrue(function ($item) {
+                            return !\in_array($item, ['rsa', 'ec']);
+                        })
+                        ->thenInvalid('The keyType %s is not valid.')
```

Comment:
I think it could be useful to add the list of available types in the message:

```
The key type "%s" is not valid. Valid types are: rsa, ec.
```

Reply:
done